src/adapters/github.ts

```diff
@@ -309,7 +309,10 @@ class GithubAdapter extends GitAdapter {
   }
 
   protected getRepositoryUrl(repo: IRepo): string {
-    return `git@${shepherdGitHubEnterpriseUrl}:${repo.owner}/${repo.name}.git`;
+    return (
+        `git@${shepherdGitHubEnterpriseUrl}:${repo.owner}/${repo.name}.git`
```

Comment:
Instead of replacing `api.`, I think we should just make the full url configurable instead of just the domain name.  I've added more details in a comment on issue #670